src/main/cli/cli.c

```diff
@@ -1410,6 +1449,26 @@ static void cliSerialPassthrough(char *cmdline)
         serialSetCtrlLineStateCb(ports[0].port, cbCtrlLine, (void *)(intptr_t)(port1PinioDtr));
     }
 
+    if (esc_sensor_passthrough) {
+        pwmOutputPort_t *pwmMotors = pwmGetMotors();
+        uint8_t motorsCount = getMotorCount();
+        for (uint8_t i = 0; i < motorsCount; i++) {
+            const ioTag_t tag = motorConfig()->dev.ioTags[i];
+            if (tag != IO_TAG_NONE) {
```

Comment:
`if (tag)` is enough, IO code was designed so that simple C test always works

Reply:
will try. thanks!

Reply:
done